react/README.md

```diff
@@ -0,0 +1,218 @@
+# Airbnb React/JSX Style Guide
+
+*A mostly reasonable approach to React and JSX*
+
+## Table of Contents
+
+  1. [Basic Rules](#basic-rules)
+  1. [Naming](#naming)
+  1. [Declaration](#declaration)
+  1. [Alignment](#alignment)
+  1. [Quotes](#quotes)
+  1. [Spacing](#spacing)
+  1. [Props](#props)
+  1. [Parentheses](#parentheses)
+  1. [Tags](#tags)
+  1. [Ordering](#ordering)
+
+## Basic Rules
+
+  - Only include one React component per file.
+  - Always use JSX syntax.
+  - Do not use `React.createElement` unless you're initializing the app from a `.js` file.
+
+## Naming
+
+  - **Extensions**: Use `.jsx` extension for React components.
```

Comment:
Can't see why *.jsx extension is necessary. *.js works perfectly. When you remove or add some component into your file, does it mean you have to rename that file? That does not sound cool for me. They are all just JS after all transpilation. 
It is like renaming your files to `.es6.js` or something.